azure-pipelines.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       echo ##vso[task.setvariable variable=VcVarsArch]%VCVARS_ARCH%
     displayName: Set VcVarsArch
   - script: |
-      git clone -q --depth 1 git://github.com/Microsoft/vcpkg.git %VCPKG_ROOT%
+      git clone -q --depth 1 git://github.com/mitza-oci/vcpkg.git %VCPKG_ROOT% -b issue9654
```

Comment:
Upstream vcpkg has been fixed

Reply:
I reversed changes in the two scripts. This script used mitza-oci originally for some reason. I changed it to Microsoft too.